From 563933799d76d8949bb1ddfea13f31ddd961bd2c Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Sun, 20 Nov 2022 20:20:30 +0800
Subject: [PATCH 05/56] Revert "drm/virtio: Use appropriate atomic state in
 virtio_gpu_plane_cleanup_fb()"

This reverts commit 1414e9bf3c307759347e6c91ed1143065ee86402.
---
 drivers/gpu/drm/virtio/virtgpu_plane.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/gpu/drm/virtio/virtgpu_plane.c b/drivers/gpu/drm/virtio/virtgpu_plane.c
index e6de62734269..6a311cd93440 100644
--- a/drivers/gpu/drm/virtio/virtgpu_plane.c
+++ b/drivers/gpu/drm/virtio/virtgpu_plane.c
@@ -213,14 +213,14 @@ static int virtio_gpu_cursor_prepare_fb(struct drm_plane *plane,
 }
 
 static void virtio_gpu_cursor_cleanup_fb(struct drm_plane *plane,
-					struct drm_plane_state *state)
+					 struct drm_plane_state *old_state)
 {
 	struct virtio_gpu_framebuffer *vgfb;
 
-	if (!state->fb)
+	if (!plane->state->fb)
 		return;
 
-	vgfb = to_virtio_gpu_framebuffer(state->fb);
+	vgfb = to_virtio_gpu_framebuffer(plane->state->fb);
 	if (vgfb->fence) {
 		dma_fence_put(&vgfb->fence->f);
 		vgfb->fence = NULL;
-- 
2.25.1

